app/Filters/SystemLogFilters.php

```diff
@@ -18,17 +18,17 @@
  */
 class SystemLogFilters extends QueryFilters
 {
-    public function type_id(int $type_id) :Builder
+    public function type_id(int $type_id): Builder
     {
         return $this->builder->where('type_id', $type_id);
     }
 
-    public function category_id(int $category_id) :Builder
+    public function category_id(int $category_id): Builder
```

Comment:
same as above, lets remove the type, and check within the method, returning early if no parameter is passed

Reply:
i did an early return, when the parameter is not isset()

Reply:
I think the problem is the type (int $type_id)

if nothing is passed into the method ie ?type_id=

the method will fail due to the type check of the input parameter.

Reply:
i reworked the methods to a string type